components/tekton/pipelineruns/build-test-suite/build-test-suite.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  labels:
+    tekton.dev/pipeline: runsuite
+  name: build-tests
+spec:
+  params:
+    - name: SRC_CONTAINERFILE
+      value: Dockerfile
```

Comment:
Is this `Dockerfile` and not `Containerfile` on purpose?

Reply:
yes, since we are using Dockerfile so far.